Ardupilog.m

```diff
@@ -59,6 +61,23 @@
             % Extract firmware version from MSG fields
             obj.findInfo();
             
+            % Attempt to find the UTC time of boot (at boot, TimeUS = 0)
+            obj.findBootTimeUTC();
+            
+            % Set the bootDatenumUTC for all LogMsgGroups
+            % HGM: This can probably be done better after some code reorganization,
+            %  but for now it works well enough. After refactoring is settled, we
+            %  might set the bootDatenumUTC when we set the LineNo, or when we store
+            %  the TimeUS data, whatever makes sense based on how we decide to handle
+            %  message filtering.
+            if ~isnan(obj.bootDatenumUTC)
+                for prop = properties(obj)'
```

Comment:
I think a better check should be
isa(obj.(prop{1}),'LogMsgGroup')

It is more clear that we want to only set the boot time for logmsggroup objects